enlace/programs/profile.php

```diff
@@ -528,7 +528,8 @@ function(response) {
         INNER JOIN Session_Names ON Participants_Programs.Program_ID=Session_Names.Session_ID
         INNER JOIN Programs ON Session_Names.Program_ID=Programs.Program_ID
         LEFT JOIN Absences ON (Program_Date_ID=Program_Date AND Participants_Programs.Participant_ID=Absences.Participant_ID)
-        WHERE Participants_Programs.Participant_ID='" . $all_p['Participant_ID'] . "' AND Absence_ID IS NULL;";
+        WHERE Participants_Programs.Participant_ID='" . $all_p['Participant_ID'] . "' AND Absences.Absent = 0;";

```

Comment:
That's actually part of a discussion with enlace.  I added the IS NULL in dosage, because I wanted to match how the old dosage reporting was done.  I'll be changing it before we push so that dosage is a percentage of recorded days, rather than total days, so I'll leave this the way it is for right now.  The concept of dosage vs attendance is weird now that they are generating a bunch of empty days.